spec/numberformat.html

```diff
@@ -239,7 +239,7 @@ <h1>Internal slots</h1>
 
       <ul>
         <li>The list that is the value of the *"nu"* field of any locale field of [[LocaleData]] must not include the values *"native"*, *"traditio"*, or *"finance"*.</li>
-        <li>[[LocaleData]].[[&lt;_locale_&gt;]] must have a [[patterns]] field for all locale values _locale_. The value of this field must be a Record, which must have fields with the names of the four number format styles: *"decimal"*, *"percent"*, *"currency"*, and *"unit"*.</li>
+        <li>[[LocaleData]].[[&lt;_locale_&gt;]] must have a [[Patterns]] field for all locale values _locale_. The value of this field must be a Record, which must have fields with the names of the four number format styles: *"decimal"*, *"percent"*, *"currency"*, and *"unit"*.</li>
         <li>
           The two fields *"currency"* and *"unit"* noted above must be Records with at least one field, *"fallback"*.
           The *"currency"* may have additional fields with keys corresponding to currency codes according to <emu-xref href="#sec-currency-codes"></emu-xref>.
```

Comment:
```suggestion
          The two fields [[Currency]] and [[Unit]] noted above must be Records with at least one field, [[Fallback]].
          The [[Currency]] Record may have additional fields with keys corresponding to currency codes according to <emu-xref href="#sec-currency-codes"></emu-xref>.
```
and so on. Note also that the change from "fallback" to "Fallback" will require fixes in [GetNumberFormatPattern](https://tc39.es/ecma402/#sec-getnumberformatpattern).